models/Product.py

```diff
@@ -24,7 +25,8 @@ class Item(EmbeddedDocument):
 class Product(Document):
     """Product Schema"""
 
-    _id = ObjectIdField(primary_key=True)
+    _id = ObjectIdField(default=ObjectId(), primary_key=True)
```

Comment:
Maybe try changing default=ObjectId() to default=ObjectId, else we'll get problems when updating/posting new documents. Since if ObjectId() doesn't change due to fast or consecutive document upsert/insert we'll be replacing the last document in the db.

Reply:
Solved, Thanks!